charts/metallb/templates/speaker.yaml

```diff
@@ -235,6 +235,9 @@ spec:
         {{- if .Values.loadBalancerClass }}
         - --lb-class={{ .Values.loadBalancerClass }}
         {{- end }}
+        {{- if .Values.speaker.wanConfig }}
+        - --ml-wan-config={{ .Values.speaker.wanConfig }}
```

Comment:
given it's a bool you can do directly =true (or, I don't remember if being a flag sets it automatically to true when the flag is specified)

Reply:
Yep, works without an explicit `true`, fixed